ThirdPartyNotices.txt

```diff
@@ -0,0 +1,684 @@
+THIRD PARTY SOFTWARE NOTICES AND INFORMATION
+
+Do Not Translate or Localize
+
+This software incorporates material from third parties. Microsoft makes certain
+open source code available at http://3rdpartysource.microsoft.com, or you may
+send a check or money order for US $5.00, including the product name, the open
+source component name, and version number, to:
+
+Source Code Compliance Team
+Microsoft Corporation
+One Microsoft Way
+Redmond, WA 98052
+USA
+
+Notwithstanding any other terms, you may reverse engineer this software to the
+extent required to debug changes to any libraries licensed under the GNU Lesser
+General Public License.
+
+_____
+
+microsoft/blingfire 0831265c1aca95ca02eca5bf1155e4251e545328 - MIT
+
+
+(c) 2018
```

Comment:
Where does these text come from?

I checked blingfire LICENSE here: https://github.com/microsoft/BlingFire/blob/master/LICENSE, which doesn't contain these lines.

Reply:
Ah yes, this is because I kept the original generated notice for blingfire. Let me update that manually as well.